.github/workflows/ci.yml

```
@@ -44,7 +44,10 @@ jobs:
           sudo apt-get update
           sudo apt-get install -y build-essential
       - name: Run tests
-        run: make test
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2046:warning:1:13: Quote this to prevent word splitting

(shellcheck)